src/test/java/pl/edu/agh/iet/katabank/steps/AccountManagementSteps.java

```diff
@@ -46,5 +42,22 @@ public AccountManagementSteps() {
                 () -> assertThat(customerAccounts)
                         .containsExactlyInAnyOrder(firstAccount, secondAccount));
 
+        Given("^a customer wants to open an account$", () -> {
+            customer = new Customer();
+        });
+
+        When("^his account is created$", () -> {
+            firstAccount = new Account(customer);
+            accountsRepository.addAccount(firstAccount);
+        });
+
+        Then("^there is a new account on his account list$", () -> {
+            customerAccounts = bank.getAccountsForCustomer(customer);
+            assertThat(customerAccounts).contains(firstAccount);
+        });
+        And("^the balance on this account is (\\d+)$", (Integer arg0) -> {
+            assertThat(firstAccount.getBalance()).isEqualByComparingTo("0");
```

Comment:
Give argument "arg0" more meaningful name.
Compare account balance with BigDecimal of value 0 rather then String.

Reply:
100% right. Corrected.